modules/machineset-yaml-alibaba-usage-stats.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating_machinesets/creating-machineset-alibaba.adoc
+
+ifeval::["{context}" == "creating-infrastructure-machinesets"]
+:infra:
+endif::[]
+
+:_content-type: REFERENCE
+[id="machineset-yaml-alibaba-usage-stats_{context}"]
+= Machine set parameters for Alibaba Cloud usage statistics
+
+The default machine sets that the installer creates for Alibaba Cloud clusters include nonessential tag values that Alibaba Cloud uses internally to track usage statistics. These tags are populated in the `securityGroups`, `tag`, and `vSwitch` parameters of the `spec.template.spec.providerSpec.value` list.
+
+When creating machine sets to deploy additional machines, you must include the required Kubernetes tags, but you choose whether to include the usage statistics tags. You can also include additional tags as needed.
```

Comment:
~~but you choose whether to include the usage statistics tags~~
Suggested:
```
but the additional usage tags will be applied by default.
```

https://github.com/openshift/cluster-api-provider-alibaba/blob/0206121348c9a0d220dd6805cea79d1eae7fd3e0/pkg/actuators/machine/instances.go#L120

Reply:
ACK, thanks, y'all!